internal/state/execution.go

```diff
@@ -416,43 +414,29 @@ func buildLastCommitInfo(block *types.Block, store Store, initialHeight int64) a
 	}
 }
 
+//TODO reword
```

Comment:
👍🏼  (since the parameters changed)

Reply:
Sorry, that TODO was on me, so let me try to finish it here as a change suggestion :-)